pkg/graveler/graveler.proto

```diff
@@ -78,8 +78,7 @@ message ImportStatusData {
   bool completed = 2;
   google.protobuf.Timestamp updated_at = 3;
   int64 progress = 4;
-  string import_branch = 5;
-  string metarange_id = 6;
-  CommitData commit = 7;
-  string error = 8;
+  string metarange_id = 5;
+  CommitData commit = 6;
+  string error = 7;
```

Comment:
Can't change field IDs - it will break compatibility

Reply:
I took a calculated risk. The import entries live only 24 hours, and the migration requires server downtime which will require to halt imports. Even in the off chance that someone tries to query an old import less than 24 hours after the import I think it's worth the risk and allow us to get rid of an unnecessary field

Reply:
It is a risk - this is the protobuf way, one field is not in use you sacrifice the field ID and mark it as deprecated.